std_msgs/Float64MultiArray tau_d

geometry_msgs/Wrench h_e
geometry_msgs/Vector3 mu_d_e

geometry_msgs/Vector3 p_d
geometry_msgs/Vector3 eps_d
geometry_msgs/Vector3 dp_d
geometry_msgs/Vector3 w_d

geometry_msgs/Vector3 p_c
geometry_msgs/Vector3 eps_c
geometry_msgs/Vector3 dp_c
geometry_msgs/Vector3 w_c